accounts/tests.py

```diff
@@ -555,3 +555,21 @@ def test_user_delete_remove_sounds(self):
         self.assertEqual(Sound.objects.filter(user__id=user.id).exists(), False)
         self.assertEqual(DeletedSound.objects.filter(user__id=user.id).exists(), True)
 
+
+class DonationTest(TestCase):
+
+    def test_donation_complete(self):
+        params = {'txn_id': '8B703020T00352816',
+                'payer_email': 'fs@freesound.org',
+                'custom': 'Anonymous',
+                'mc_currency': 'USD',
+                'mc_gross': '1.00'}
+
+        with mock.patch('accounts.views.requests') as mock_requests:
+            mock_response = mock.Mock(text='VERIFIED')
+            mock_requests.post.return_value = mock_response
+            resp = self.client.post(reverse('donation-complete'), params)
+        self.assertEqual(resp.status_code, 200)
```

Comment:
Mmm, I disagree: If the client.post fails silently, resp might be None and resp.status_code won't exist